### PR TITLE
RPC-147 delete filled order fill

### PR DESCRIPTION
**Pull request by SKonstantinS**

### CODE REVIEW

File: exchange-v2/test/v2/ExchangeV2.rarible.test.js [hunk 1]
```diff
@@ -62,33 +62,33 @@ contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
 	beforeEach(resetState);
 
 	describe("on-chain orders", () => {
-        it("should create, update then cancel order", async () => {
```
Comment (disenotov) [inline exchange-v2/test/v2/ExchangeV2.rarible.test.js:65]: it would be better to fix formatting in separate PRs

File: exchange-v2/contracts/ExchangeV2Core.sol [hunk 2]
```diff
@@ -243,12 +243,19 @@ abstract contract ExchangeV2Core is Initializable, OwnableUpgradeable, AssetMatc
...
 
-        uint takeValueLeft = order.takeAsset.value.sub(getOrderFill(order, hash));
+        uint value;
+        if (dataOrder.isMakeFill) {
+            value = order.makeAsset.value;
+        } else {
+            value = order.takeAsset.value;
+        }
+
+        uint takeValueLeft = value.sub(getOrderFill(order, hash));
```
Comment (disenotov) [inline exchange-v2/contracts/ExchangeV2Core.sol:258]: deleteFilledOrder() can take fills of orders as argument (LibFill.FillResult memory newFill from matchAndTransfer() ) so we don't read storage 2 times

File: exchange-v2/test/v2/ExchangeV2.rarible.test.js (hunk 1 above)
Comment (SKonstantinS) [reply exchange-v2/test/v2/ExchangeV2.rarible.test.js:65]: i`ll check it in RPC-143-Audit-OnChainOrders after merging all branches

File: exchange-v2/contracts/ExchangeV2Core.sol (hunk 2 above)
Comment (SKonstantinS) [reply exchange-v2/contracts/ExchangeV2Core.sol:258]: method getFillSetNew() read **fills** and remember it in return value, after change **fills**
getOrderFill() don`t need old **fills**, it use new one
so there are a lot of broken tests if use LibFill.FillResult memory newFill from matchAndTransfer()

**Comment by SKonstantinS** [comment]
OnChainOrders have to be refactoring, now close this PR
